code/modules/cooking/trays.dm

```diff
@@ -0,0 +1,275 @@
+/*
+ * Trays - Nanako, ported by Jade
```

Comment:
What kind of updates were included in this trays.dm ? It looks like you're getting rid of our updated version for an older one.

Reply:
It uses vis_contents now, meaning you can pick items up directly off trays.